app/components/works/keywords_row_component.rb

```diff
@@ -0,0 +1,27 @@
+# typed: false
+# frozen_string_literal: true
+
+module Works
+  # Allows the user to search for keywords or provide freetext keywords
+  class KeywordsRowComponent < ApplicationComponent
+    def initialize(form:, key:)
+      @form = form
+      @key = key
```

Comment:
Is this parameter necessary?

Reply:
No - doesn't seem to be. I've dropped the key parameters in this PR.